node/resolvers/catalog/productSearch.ts

```diff
@@ -30,14 +30,15 @@ export const resolvers = {
       _: any,
       { vtex: { account }, clients: { catalog } }: Context
     ) => {
-      const query = translatedArgs.query || ''
-      const map = translatedArgs.map || ''
+      const query = translatedArgs.query ?? ''
+      const map = translatedArgs.map ?? ''
       const queryAndMap = zip(
         query.toLowerCase().split('/').map(decodeURIComponent),
         map.split(',')
       )
 
       const categoriesSearched = queryAndMap
+        // eslint-disable-next-line no-shadow
         .filter(([_, m]) => m === 'c')
```

Comment:
`_` is already the name of the first argument of the parent function, but underscore is a correct rule, so you can call as many as you like.

In this case, only two are enough.

```suggestion
        .filter(([__, m]) => m === 'c')
```